### PR TITLE
Migrate amp-story-consent away from position: fixed.

### LINKED ISSUE #15057: Migrate amp-story-consent away from position: fixed
See: #15028

### PR DESCRIPTION
**Pull request by gmajoulet**
Fixes #15057

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story.css [hunk 1]
```diff
@@ -23,6 +23,16 @@ amp-story, amp-story-page, amp-story-grid-layer, amp-story-cta-layer {
...
 }
 
+amp-consent {
+  position: absolute !important;
+  top: 0 !important;
+  left: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: none !important;
+  z-index: initial !important;
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story.css]: Don't we want to enforce that this is on top of stuff?

Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story.css]: The thoughts behind this were to do that at the component (``amp-story-consent``) level.
Setting the ``z-index`` to ``initial`` ensures it creates no stacking context, so it's the ``amp-story-consent`` z-index that actually matters.

Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story.css]: (I could see myself updating the z-index in the amp-story-consent.css file later, raging for 30 minutes before figuring out I actually have to change the amp-consent z-index in the amp-story.css file 🙄)

Comment (newmuis) [reply extensions/amp-story/1.0/amp-story.css]: Will that work?  If `amp-story-consent` stays nested within `amp-consent`, then the `amp-consent` z-index is what will matter for comparing it to `amp-consent`'s siblings (`amp-story-page` elements).

**Pull request merged**